service/src/test/java/com/commercetools/util/HttpRequestUtilParallelTest.java

```diff
@@ -83,7 +141,20 @@ private void makeParallelRequests(final int nThreads, final int requests, final
                 .withFailMessage(format("The execution thread pool was not able to shutdown in time %s msec", maxCriticalPathDurationMsec))
                 .isTrue();
 
-        assertThat(counter.get()).isEqualTo(requests);
+        assertThat(counter.get())
+                .withFailMessage(format("Incorrect number of finished requests: expected %d, executed %d. " +
+                        "See the standard error logs (stderr) for exceptions", counter.get(), requestsNumber))
```

Comment:
`Please see`

Reply:
No, "**See, motherXXX, the logs!**", otherwise you and all your family will die after the ring!

Reply:
455f9258